charts/kubearchive/templates/role.yaml

```
@@ -23,3 +23,15 @@ rules:
     {{- end }}
   {{- end }}
 {{- end }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ .Values.apiServer.clusterRole }}
```

Call everything the same, so its easier tracking it.

```suggestion
  name: {{ .Values.apiServer.name }}
```